woocommerce-admin.php

```diff
@@ -15,305 +15,302 @@
+		require_once WC_ADMIN_ABSPATH . 'includes/notes/class-wc-admin-notes-mobile-app.php';
+		require_once WC_ADMIN_ABSPATH . 'includes/notes/class-wc-admin-notes-welcome-message.php';
 	}
-}
-add_action( 'admin_menu', 'wc_admin_devdocs' );
 
-/**
- * Daily events to run.
- *
- * Note: WC_Admin_Notes_Order_Milestones::other_milestones is hooked to this as well.
- */
-function wc_admin_do_wc_admin_daily() {
-	WC_Admin_Notes_New_Sales_Record::possibly_add_sales_record_note();
-	WC_Admin_Notes_Giving_Feedback_Notes::add_notes_for_admin_giving_feedback();
-	WC_Admin_Notes_Mobile_App::possibly_add_mobile_app_note();
-}
-add_action( 'wc_admin_daily', 'wc_admin_do_wc_admin_daily' );
+	/**
+	 * Filter in our ActionScheduler Store class.
+	 *
+	 * @param string $store_class ActionScheduler Store class name.
+	 * @return string ActionScheduler Store class name.
+	 */
+	public function replace_actionscheduler_store_class( $store_class ) {
+		// Don't override any other overrides.
+		if ( 'ActionScheduler_wpPostStore' !== $store_class ) {
+			return $store_class;
+		}
+
+		// Include our store class here instead of wc_admin_plugins_loaded()
+		// because ActionScheduler is hooked into `plugins_loaded` at a
+		// much higher priority.
+		require_once WC_ADMIN_ABSPATH . '/includes/class-wc-admin-actionscheduler-wppoststore.php';
+
+		return 'WC_Admin_ActionScheduler_WPPostStore';
+	}
+
+	/**
+	 * Removes core hooks in favor of our local feature plugin handlers.
+	 *
+	 * @see WC_Admin_Library::__construct()
+	 */
+	protected function hooks() {
+		remove_action( 'init', array( 'WC_Admin_Library', 'load_features' ) );
+		remove_action( 'admin_enqueue_scripts', array( 'WC_Admin_Library', 'register_scripts' ) );
+		remove_action( 'admin_enqueue_scripts', array( 'WC_Admin_Library', 'load_scripts' ), 15 );
+		remove_action( 'woocommerce_components_settings', array( 'WC_Admin_Library', 'add_component_settings' ) );
+		remove_filter( 'admin_body_class', array( 'WC_Admin_Library', 'add_admin_body_classes' ) );
+		remove_action( 'admin_menu', array( 'WC_Admin_Library', 'register_page_handler' ) );
+		remove_filter( 'admin_title', array( 'WC_Admin_Library', 'update_admin_title' ) );
+
+		remove_action( 'rest_api_init', array( 'WC_Admin_Library', 'register_user_data' ) );
+		remove_action( 'in_admin_header', array( 'WC_Admin_Library', 'embed_page_header' ) );
+		remove_filter( 'woocommerce_settings_groups', array( 'WC_Admin_Library', 'add_settings_group' ) );
+		remove_filter( 'woocommerce_settings-wc_admin', array( 'WC_Admin_Library', 'add_settings' ) );
+
+		remove_action( 'admin_head', array( 'WC_Admin_Library', 'update_link_structure' ), 20 );
+
+		/*
+		* Remove the emoji script as it always defaults to replacing emojis with Twemoji images.
+		* Gutenberg has also disabled emojis. More on that here -> https://github.com/WordPress/gutenberg/pull/6151
+		*/
+		remove_action( 'admin_print_scripts', 'print_emoji_detection_script' );
```

Comment:
I'm thinking this hook removal actually belongs in `WC_Admin_Loader` so we run it in core too. I must have missed it previously.